app/javascript/mastodon/actions/importer/normalizer.js

```diff
@@ -36,8 +36,14 @@ export function normalizeStatus(status, normalOldStatus) {
     normalStatus.poll = status.poll.id;
   }
 
-  if (status.card?.author_account) {
-    normalStatus.card = { ...status.card, author_account: status.card.author_account.id };
+  if (status.card) {
+    normalStatus.card = {
+      ...status.card,
+      authors: status.card.authors.map(author => ({
+        ...author,
+        account: author.account?.id,
```

Comment:
I would prefer to use `accountId` as the key here, to make it explicit that its not a full account

Reply:
Even though elsewhere we don't do that?

Reply:
I started to use this pattern when reworking the state. It can be done later, when I get to converting the `Status` to the modern state, but I assumed that it might be easier to do it when introducing it.